webapp/platform/types/src/users.ts

```diff
@@ -20,6 +21,7 @@ export type UserNotifyProps = {
     channel: 'true' | 'false';
     mention_keys: string;
     desktop_notification_sound?: 'Bing' | 'Crackle' | 'Down' | 'Hello' | 'Ripple' | 'Upstairs';
+    calls_notification_sound?: 'Dynamic' | 'Calm' | 'Urgent' | 'Cheerful';
```

Comment:
If we follow the existing convention should this be `calls_desktop_notification_sound` or `desktop_calls_notification_sound`? Otherwise we may as well drop `desktop` altogether but it depends on the answer to my previous question I guess.

Reply:
At first I chose this because `calls_desktop_notification_sound` was just too long. But now (see: https://community.mattermost.com/core/pl/uqz5m6czqtbquy384goyq6ifie) I think the consensus is that the sound will be shared across desktop/web and mobile, so the name reflects that it's for both. If we think we might have a separate one for mobile in the future, we could name the mobile one `calls_notification_sound_mobile` to differentiate (if it's ever needed). 
tldr: imo, keep this and if we do need a separate mobile, make that one a more specific name.

Reply:
Sounds good, thanks for confirming 👍